core/ui/compose/common/src/main/kotlin/app/k9mail/core/ui/compose/common/mvi/BaseViewModel.kt

```diff
@@ -54,4 +56,11 @@ abstract class BaseViewModel<STATE, EVENT, EFFECT>(
             _effect.emit(effect)
         }
     }
+
+    protected fun handleOneTimeEvent(event: EVENT, block: () -> Unit) {
```

Comment:
I would like this to be tested as it is a base component and also alter the documentation to document on-off event handling.